backend/src/main/java/com/bakdata/conquery/models/forms/arx/ARXConfig.java

```diff
@@ -0,0 +1,107 @@
+package com.bakdata.conquery.models.forms.arx;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.regex.Pattern;
+
+import com.bakdata.conquery.commands.ManagerNode;
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.io.cps.CPSTypeIdResolver;
+import com.bakdata.conquery.io.freemarker.Freemarker;
+import com.bakdata.conquery.io.jackson.Jackson;
+import com.bakdata.conquery.models.config.PluginConfig;
+import com.bakdata.conquery.models.forms.arx.models.KAnonymity;
+import com.bakdata.conquery.models.forms.arx.models.PopulationUniqueness;
+import com.bakdata.conquery.models.forms.arx.models.PrivacyModel;
+import com.bakdata.conquery.models.forms.frontendconfiguration.FormFrontendConfigInformation;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableCollection;
+import io.dropwizard.views.View;
+import io.github.classgraph.Resource;
+import io.github.classgraph.ResourceList;
+import lombok.Data;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.deidentifier.arx.risk.RiskModelPopulationUniqueness;
+
+@CPSType(id = "ARX", base = PluginConfig.class)
+@Data
+@Slf4j
+public class ARXConfig implements PluginConfig {
+
+	private TreeMap<String, PrivacyModel> privacyModels = new TreeMap<>(Map.of(
```

Comment:
```suggestion
	private Map<String, PrivacyModel> privacyModels = new TreeMap<>(Map.of(
```

Oder TreeMap hier wichtig?

Reply:
Ich habe hier TreeMap genommen, damit die Werte gleich alphabetisch im Frontend ankommen